website/src/site/site.xml

```diff
@@ -51,7 +51,7 @@ under the License.
     <href></href>
   </bannerLeft>
   <bannerRight>
-    <src>apache-feather.gif</src>
+    <src>apache-feather.png</src>
```

Comment:
Uh oh, I will have to setup myself for website merges again. Any other volunteers?